Rdkafka.xs

```diff
@@ -288,6 +288,74 @@ krd_flush(rdk, timeout_ms)
     CODE:
         rd_kafka_flush(rdk->rk, timeout_ms);
 
+int
+krd_producev(rdk, params)
+        rdkafka_t* rdk
+        HV* params
```

Comment:
can we add timestamp and default it to one of the macros? RD_KAFKA_TIMESTAMP_CREATE_TIME maybe?

Reply:
Something, either `librdkafka` or the broker, is already adding the current time as the `timestamp` with the type `RD_KAFKA_TIMESTAMP_CREATE_TIME` which corresponds to "Message creation time". I confirmed this by consuming a message produced with `librdkafka`. I'm not sure whether it's the library or the broker adding this, although I would presume `RD_KAFKA_TIMESTAMP_CREATE_TIME` means it's the library. I will investigate.

Reply:
It's the library in `rdkafka_msg.c`:

```
        if (timestamp)
                rkm->rkm_timestamp = timestamp;
        else
                rkm->rkm_timestamp = rd_uclock() / 1000;
        rkm->rkm_tstype = RD_KAFKA_TIMESTAMP_CREATE_TIME;
```

Reply:
I believe the broker adds it, as $msg->timestamp always has a valuue. This is only if we want to send a custom value when we produce messages. RD_KAFKA_TIMESTAMP_CREATE_TIME is probably the most reasonable default

Reply:
In any case this request is not a blocker